mautrix_telegram/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "0.10.2-mod-6"
+__version__ = "0.11.0"
```

Comment:
We can't use the mod syntax here because it breaks the setup script, as PIP440 doesn't allow for our version format.